src/main/java/com/cloudbees/jenkins/plugins/BitbucketJobProbe.java

```diff
@@ -65,6 +68,15 @@ public void triggerMatchingJobs(String user, String url, String scm) {
         }
     }
 
+    private boolean hasBeenTriggered(List<SCM> scmTriggered, SCM scmTrigger) {
+        for (SCM scm : scmTriggered) {
+            if (scm.equals(scmTrigger)) {
```

Comment:
Does this actually work? At least `MercurialSCM` does not override `Object.equals`, so I suspect not. There is no test here to prove anything.